README.md

```diff
@@ -1,13 +1,14 @@
-# Jenkins UI sample plugin
+![Jenkins Design Library](logo-dark.svg#gh-dark-mode-only)
```

Comment:
This seems to be linking to https://github.com/jenkinsci/ui-samples-plugin/blob/a4dc40e5c59e22e675f897e7ec6747a8d2633970/logo-dark.svg#gh-dark-mode-only

When I click it, is there a way to make it not a link?

Reply:
Had a Google but nothing helpful comes up sadly, there's a hacky way to link it to `#` so it won't navigate to another page but it isn't ideal.

Reply:
That’s fine then